tests/integration-tests/tests/storage/test_efs.py

```diff
@@ -80,27 +82,35 @@ def test_multiple_efs(
 
     To verify the efs is the existing efs, the test expects a file with random ran inside the efs mounted
     """
-    existing_efss = []
+    # Names of files that will be written from separate instance. The test checks the cluster nodes can access them.
     existing_efs_filenames = []
     existing_efs_mount_dirs = []
-    num_existing_efs = 2
-    for i in range(num_existing_efs):
-        efs_id = efs_stack_factory().cfn_resources["FileSystemResource"]
-        existing_efss.append(efs_id)
-        existing_efs_filenames.append(
-            _write_file_into_efs(region, vpc_stack, efs_id, request, key_name, cfn_stacks_factory)
+    if request.config.getoption("benchmarks") and os == "alinux2":
```

Comment:
Is it possible to refactor the code to have all the "benchmarks" related code into a separate function?
I mean I'd avoid calling `if request.config.getoption("benchmarks")` in multiple places.

In this way it would be easier to understand what to modify or remove to affect only benchmarks related code.

Reply:
ok